lib/gemfury/command/app.rb

```diff
@@ -37,10 +39,13 @@ def list
   desc "versions NAME", "List all the package versions"
   def versions(gem_name)
     with_checks_and_rescues do
-      versions = client.versions(gem_name)
+      package = client.package(gem_name)
+      versions = package['versions']
       shell.say "\n*** #{gem_name.capitalize} Versions ***\n\n"
       versions.each do |v|
-        shell.say v['version']
+        size = '%s bytes' % v['size']
+        uploaded = Time.parse(v['updated_at']).strftime('%F %R %z')
+        shell.say "#{v['version']} (#{size}, #{uploaded})"
```

Comment:
Is there a Thor or Highline feature to print things aligned in table-like form, rather than doing parentheses?